macroslib/src/java_jni/jni-include.rs

```diff
@@ -193,7 +193,7 @@ struct JniEnvHolder<'a> {
 }
 
 #[allow(dead_code)]
-impl<'a> Drop for JniEnvHolder<'a> {
```

Comment:
This fixes "warning: the following explicit lifetimes could be elided: 'a" caused by https://rust-lang.github.io/rust-clippy/master/index.html#missing_safety_doc 
https://rust-lang.github.io/rust-clippy/master/index.html#needless_lifetimes